src/tools/csg_resample.cc

```diff
@@ -171,7 +171,7 @@ int main(int argc, char **argv) {
           if (in.x(i) < sp_min) {
```

Comment:
Do we need to do the same for `minindex`?

Reply:
Really not sure about it..